OctoAwesome/OctoAwesome.Basics/Entities/WauziEntity.cs

```diff
@@ -1,59 +1,75 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using engenious;
-using OctoAwesome.Basics.EntityComponents;
-using OctoAwesome.EntityComponents;
-
+﻿using engenious;
+using OctoAwesome.Entities;
 namespace OctoAwesome.Basics.Entities
 {
-    public class WauziEntity : UpdateableEntity
+    public class WauziEntity : Entity, IControllable, OctoAwesome.Entities.IDrawable
     {
-        public int JumpTime { get; set; }
-
-        public WauziEntity() : base()
+        class WauziTestController : IEntityController
         {
+            public float Tilt { get; set; }
+            public float Yaw { get; set; }
+            public Vector3 Direction { get; set; }
+            public Index3? SelectedBlock { get; set; }
+            public Vector2? SelectedPoint { get; set; }
+
+            public OrientationFlags SelectedSide => OrientationFlags.None;
+            public OrientationFlags SelectedEdge => OrientationFlags.None;
+            public OrientationFlags SelectedCorner => OrientationFlags.None;
+
+            public bool InteractInput { get; set; }
+            public bool ApplyInput { get; set; }
+            public bool JumpInput { get; set; }
+
+            public bool[] SlotInput{ get; } = new bool[10];
+
+            public bool SlotLeftInput { get; set; }
+            public bool SlotRightInput { get; set; }
+
+            public WauziTestController()
+            {
+            }
         }
 
+        public IEntityController Controller => currentcontroller;
+        public bool DrawUpdate => true;
+        // TODO: Auslagern in die Definition
+        public string Name => "Wauzi";
+        public string ModelName => "dog";
+        public string TextureName => "texdog";
+        public float BaseRotationZ => -90f;
+
+        public float Height => 1;
+        public float Radius => 1;
+        private IEntityController currentcontroller;
+        private float jumptime;
+        public WauziEntity() : base(true)
```

Comment:
Eigenschaften nicht änderbar

Reply:
Das ist die konkrete Implementierung einer einer Entity für Testzwecke die außerdem in Entwicklung ist.